modules/feincms-pages/scripts/init.js

```diff
@@ -7,8 +7,7 @@
         'feincms-pages.page-group',
         'feincms-pages.load-page',
         'feincms-pages.constants',
-        'feincms-pages.routes',
-        'feincms-pages.templates'
+        'feincms-pages.routes'
```

Comment:
Initially most packages we developed weren't depending on a templates module, because doing so made the assumption that templates would be provided from a compiled JS file using the `$templateCache`. 

Loading the templates separately allowed for the case where we simply provided templates via http javascript calls.

In practice however, we are never doing that, so we've started to specifically define template module dependencies to make sure they are not forgotten. It's not a super critical issue either way, but since this PR is specifically about adding linting, I'd say don't change the existing functionality for this PR. The this question doesn't block this work and then we can explore adding template dependencies in another PR.